rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,37 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+All dxw-owned respositories MUST use a standard linting tool for the
+main programming language the repo is written in, where one is available.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked
+this can lead to inconsistencies in style, which over time can make code
+messy, and difficult to maintain and review. It can also lead to bikeshedding
+over code style in code reviews, which can cause conflict and slow down code
+review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories MUST use a
+standard linting tool to make sure code meets a standard style. Examples
+of this include (but are not limited to):
+
+- [Standard.rb](https://github.com/testdouble/standard)
+- [ESLint](https://eslint.org/)
```

Comment:
In PHP land we use php-cs-fixer and psalm simultaneously (php-cs-fixer checks style and psalm checks for type safety):

https://github.com/FriendsOfPHP/PHP-CS-Fixer
https://github.com/vimeo/psalm